### PR TITLE
Add option --msaa-sample-count

### PR DESCRIPTION
**Pull request by gyagp**
MSAA sample count is no longer fixed in 4. Instead, its value can be
designated by option, with default value as 4.

### CODE REVIEW

**Comment by gyagp** [comment]
@hujiajie, PTAL!

**Comment by hujiajie** [comment]
@gyagp Thanks for the patch!

To summarize the bot failure, `refs/pull/origin/<pr>/merge` is assumed to be the triggering commit in our CI scripts, but GitHub creates a separate commit this time. I don't know if it's intentional. Can you `git commit --amend` your patch and `git push -f` again?

And I'd like to mention the existing `--enable-alpha-blending` option. I hope to have consistent command-line syntax here, so either you need to update your patch, or we will change the other next.

**Comment by gyagp** [comment]
This is a good point. I removed --enable-msaa option and unified to --msaa-sample-count (1 means no MSAA). For option "--enable-alpha-blending", I think we also need to change its name to "--alpha-blending", while 1 means opaque.

File: src/include/CmdArgsHelper.h [hunk 1]
```diff
@@ -16,7 +16,7 @@ const char *cmdArgsStrAquarium = R"(Options and arguments:
 --discrete-gpu          : Choose discrete gpu to render the application. This is only supported on Dawn and D3D12 backend.
 --enable-alpha-blending=[0, 1] | false : Force enable alpha blending to a specific value or disable alpha blending for all models. By default, alpha blending is enabled.
 --enable-instanced-draws : specifies rendering fishes by instanced draw. By default, fishes are rendered by individual draw.Instanced rendering is only supported on dawn and d3d12 backend now.
---enable-msaa           : Enable 4 samples MSAA. MSAA of angle backend is not supported now.
+--msaa-sample-count     : specifies sample count of MSAA, while 1 for non-MSAA.
```
Comment (hujiajie) [inline src/include/CmdArgsHelper.h]: Maybe `--msaa-sample-count [count]`? (and we should convert all `[]` to `<>` in the future)

**Review by hujiajie — COMMENTED** [review]
Almost LGTM, please also update README.md accordingly.

File: src/include/CmdArgsHelper.h (hunk 1 above)
Comment (gyagp) [reply src/include/CmdArgsHelper.h]: May I use the next patch to change the way we do arg parser now? I hope to introduce something like "argparse" in python.

Comment (hujiajie) [reply src/include/CmdArgsHelper.h]: Sounds good, but I still hope README and the help message here can be updated.

**Comment by gyagp** [comment]
Done

**Comment by hujiajie** [comment]
Thanks! Merging

**Pull request merged**